### DISCUSSION TITLE
Could support be added for ARM64 and ARM32?

### DISCUSSION THREAD

**Discussion by ghost** [oven-sh/bun#681] [Ideas]
Hi, I really want to use this engine, but I cannot compile it on the Raspi Pi, Neither are there supported formats of a pre-compiled version of Bun. Perhaps a build process could be added for this (I literally no nothing about zig so I don't know how to make this happen successfully)

I'm grateful if this is addressed or planned.

**Comment by moderation** [comment]
I don't believe `arm32` will ever be supported. `arm64` for Linux is being discussed at https://github.com/oven-sh/bun/issues/75 and `arm64` for MacOS Apple Silicon is already supported. Fortunately there are 64 bit Linux distros for the Raspberry PI

**Reply by ghost** [reply]
Oh ok. I tried searching for arm64 and found nothing
